packages/@react-spectrum/picker/stories/Picker.stories.tsx

```diff
@@ -55,530 +70,255 @@ let withSection = [
-        <Item key="Three">Three</Item>
-      </Picker>
-    )
-  )
-  .add(
-    'defaultSelectedKey (uncontrolled)',
-    () => (
-      <Picker label="Test" defaultSelectedKey="One" onSelectionChange={action('selectionChange')}>
-        <Item key="One">One</Item>
-        <Item key="Two">Two</Item>
-        <Item key="Three">Three</Item>
-      </Picker>
-    )
-  )
-  .add(
-    'picker closes on blur',
-    () => (
-      <>
-        <div style={{display: 'flex', width: 'auto', margin: '250px 0'}}>
-          <label htmlFor="focus-before">Focus before</label>
-          <input id="focus-before" />
-          <Picker label="Test" defaultSelectedKey="One" onSelectionChange={action('selectionChange')}>
-            <Item key="One">One</Item>
-            <Item key="Two">Two</Item>
-            <Item key="Three">Three</Item>
-          </Picker>
-          <label htmlFor="focus-after">Focus after</label>
-          <input id="focus-after" />
-        </div>
-      </>
-    )
-  )
-  .add(
-    'isLoading',
-    () => (
-      <Picker label="Test" isLoading items={[]}>
-        {item => <Item>{item.name}</Item>}
-      </Picker>
-    )
-  )
-  .add(
-    'isLoading, isQuiet',
-    () => (
-      <Picker label="Test" isLoading isQuiet items={[]}>
-        {item => <Item>{item.name}</Item>}
-      </Picker>
-    )
-  )
-  .add(
-    'isLoading more',
-    () => (
-      <Picker label="Test" isLoading items={flatOptions}>
+      )
+    ),
+    items: withSection
+  },
+  name: 'dynamic with sections'
+};
+
+export type ComplexItemsStory = ComponentStoryObj<typeof ComplexItemsPicker>;
+export const ComplexItems: ComplexItemsStory = {
+  render: (args) => <ComplexItemsPicker {...args} />,
+  name: 'complex items'
+};
+
+export const LongItemText: PickerStory = {
+  args: {
+    children: (item: any) => <Item key={item.key}>{item.name}</Item>,
+    items: longItemText
+  },
+  name: 'long item text'
+};
+
+export const FalsyKey: PickerStory = {
+  args: {
+    children: (item: any) => <Item key={item.key}>{item.name}</Item>,
+    items: falsyKey
+  },
+  name: 'falsy item key'
+};
+
+export const NoLabel: PickerStory = {
+  args: {
+    children: (item: any) => <Item>{item.name}</Item>,
+    items: flatOptions,
+    'aria-label': 'Test',
+    label: null
+  },
+  name: 'no visible label'
+};
+
+export const ContextualHelpPicker: PickerStory = {
+  args: {
+    children: (item: any) => <Item>{item.name}</Item>,
+    items: flatOptions,
+    contextualHelp: (
+      <ContextualHelp>
+        <Heading>What is a segment?</Heading>
+        <Content>Segments identify who your visitors are, what devices and services they use, where they navigated from, and much more.</Content>
+      </ContextualHelp>
+    )
+  },
+  name: 'contextual help'
+};
+
+export const ControlledOpen: PickerStory = {
+  args: {
+    children: (item: any) => <Item>{item.name}</Item>,
+    items: flatOptions,
+    isOpen: true
+  },
+  name: 'isOpen (controlled)'
+};
```

Comment:
any reason not to have this be driven by a control?

Reply:
no particular reason, just didn't feel as behaviorally/visually useful as a control but I'm happy to add